commcare_export/commcare_hq_client.py

```diff
@@ -138,7 +139,7 @@ def iterate_resource(resource=resource, params=params):
                     raise ResourceRepeatException("Requested resource '{}' {} times with same parameters".format(resource, repeat_counter))
 
                 batch = self.get(resource, params)
-                last_params = params
+                last_params = copy.copy(params)
```

Comment:
don't we use deepcopy?

Reply:
i guess it'd be safer, but the object contains url parameters which are both not nested and have primitive values so i don't think it's necessary.

Reply:
cool, approving it

Reply:
Where is `params` mutated? I don't see it.

nit: `dict` has a copy method
```suggestion
                last_params = params.copy()
```

Reply:
It turns out that the return value of paginator.next_page_params_from_batch is a member variable called payload, so after assigning the return value once to params and then assigning to last_params, those two variables would remain aliases to the same dictionary.

Reply:
@millerdev I'm not 100% convinced that `params` will ever not be `None` so refraining form making that change for now. @solleks feel free to update / clean up as you see fit. My priority was getting a release out that did not have this bug in it as quickly as possible.